.github/workflows/build.yml

```diff
@@ -14,14 +14,16 @@ jobs:
       fail-fast: false
 
       matrix:
-        os: [macos-latest]
+        os: [ubuntu-latest]
```

Comment:
Can we build both ubuntu and macOS? I don't want to break the Mac build.

Also, let's always disable runtime on Mac and always enable runtime on ubuntu, I think that'll give good enough build coverage.

Reply:
Also runtime is only officially supported on ubuntu 20.04, whereas latest is 22.04

Reply:
@vmilosevic should we add a test for ubuntu 20.04 as well?

Reply:
Just double check if metal uses ubuntu-latest.  Let's copy whatever ubuntu version they're running on in CI.  I want to avoid chasing down environment issues.

Reply:
Currently tt-metal run on 20.04

```
matrix:
        build: [
          {type: Debug, runs-on: ubuntu-20.04},
          {type: RelWithDebInfo, runs-on: build}
        ]
        arch: [grayskull, wormhole_b0, blackhole]
        os: [ubuntu-20.04]
```